packages/table/test/editableCellTests.tsx

```diff
@@ -126,4 +126,16 @@ describe("<EditableCell>", () => {
         elem.find("input").simulate("blur");
         expect(onChangeSpy.firstCall.args).to.deep.equal([CHANGED_VALUE, ROW_INDEX, COLUMN_INDEX]);
     });
+
+    it("defaults to no wrapText", () => {
+        const elem = mount(<EditableCell />);
+
+        expect(elem.find(`.${Classes.TABLE_NO_WRAP_TEXT}`).children().length).to.equal(1);
```

Comment:
actually this doesn't do what you think it does: https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#children-now-has-slightly-different-meaning. 

i think this test can be simply `expect(elem.find(`.${Classes.TABLE_NO_WRAP_TEXT}`).exists()).to.be.true;` and the other one `.to.be.false`

Reply:
the `.exists()` check is very intuitive: no wrap text means the NO_WRAP_TEXT element exists.

Reply:
@face can I get a push from you real quick to resolve this?